llvm/include/llvm/ObjectYAML/DWARFYAML.h

```diff
@@ -118,6 +118,28 @@ struct Unit {
   std::vector<Entry> Entries;
 };
 
+struct IdxForm {
+  llvm::dwarf::Index Idx;
+  llvm::dwarf::Form Form;
+};
+
+struct DebugNameAbbreviation {
+  llvm::yaml::Hex64 Code;
+  llvm::dwarf::Tag Tag;
+  std::vector<IdxForm> Indices;
+};
+
+struct DebugNameEntry {
+  llvm::yaml::Hex32 NameStrp;
+  llvm::yaml::Hex64 Code;
+  std::vector<llvm::yaml::Hex64> Values;
+};
+
+struct DebugNamesSection {
+  std::vector<DebugNameAbbreviation> Abbrevs;
```

Comment:
should this be <const DebugNameAbbreviation\>?

Reply:
that would disable the compiler's ability to move objects in the vector when it needs to resize https://godbolt.org/z/8hT71Khq5

Reply:
Ah didn't think of that.